README-kor.adoc

```diff
@@ -0,0 +1,321 @@
+// Copyright 2019-2022 The Khronos Group, Inc.

```

Comment:
this file should be in `lang/README-kor.adoc`

Reply:
seems there is already one and this is a duplicate

Reply:
I deleted the duplicate files, leaving only the lang/README-kor.adoc file.